clouddriver-openstack/src/main/resources/asg.yaml

```diff
@@ -0,0 +1,79 @@
+heat_template_version: 2015-04-30
```

Comment:
Good question - I think for now we want to provide 'the standard' way of being able to configure an autoscaling group, with the hardcoded templates. Later on add in the feature to provide custom templates.

Overriding the default via configuration is a good idea, as long as people do their diligence to honor the properties that the template can expect to be given to it. The Openstack API is finicky when properties are provided but not used, and doesn't work at all when properties are not provided but expected.

Being able to pass in the template via free-form text is also a good idea, but again it would need to honor the expected parameters. We could use the freeform details field to store the template string.

Added a comment in the code per ^^^.